crates/bevy_render/src/lib.rs

```diff
@@ -273,6 +273,11 @@ impl Plugin for RenderPlugin {
                         .get_stage_mut::<SystemStage>(&RenderStage::Cleanup)
                         .unwrap();
                     cleanup.run(&mut render_app.world);
+                }
+                {
+                    #[cfg(feature = "trace")]
+                    let _stage_span =
```

Comment:
@bjorn3 I've left this in for now, as I'm struggling to find a compelling argument for running `Cleanup` and clearing the entities to have no differentiation. Perhaps it should be nested, but that's a different question to what you were requesting.